### ISSUE TITLE
Title element in plugin configuration ignored for maven plugin type projects

### ISSUE THREAD

**Issue by dgalanp** [bsorrentino/maven-confluence-plugin#102]
When I try to publish a page in confluence with the following plugin configuration:
```xml
          <plugin>
            <groupId>org.bsc.maven</groupId>
            <artifactId>confluence-reporting-maven-plugin</artifactId>
            <executions>
              <execution>
                <id>default-site-deploy</id>
                <goals>
                  <goal>deploy</goal>
                </goals>
                <phase>site-deploy</phase>
              </execution>
            </executions>
            <configuration>
              <endPoint>${confluence.home}/rpc/xmlrpc</endPoint>
              <spaceKey>${confluence.space.key}</spaceKey>
              <parentPageTitle>${project.confluence.report.parent}</parentPageTitle>
              <title>${project.name}</title>
              <templateWiki>src/site/confluence/info.confluence</templateWiki>
              <wikiFilesExt>.confluence</wikiFilesExt>
              <serverId>my-confluence-server</serverId>
            </configuration>
          </plugin>
```

In a project which happens to be a maven plugin, the "title" element in the configuration is ignored and the page title is of the form:  "project-name-1.2.3.4"

Also the page contains a message:
```
auto generated page
this page has been generated by plugin: org.bsc.maven:maven-confluence-reporting-plugin
```

**Comment by bsorrentino** [comment]
Hi i've worked on issue. I have release a ```4.8-SNAPSHOT``` with fix

Please,  could you test it and let me know ?

Thanks in advance

**Comment by dgalanp** [comment]
Wow! That was fast!  :)

It seems to be working now:
  - Title is as expected
  - Warning disappeared

Thanks!

**Comment by bsorrentino** [comment]
fix release on 4.8
